cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -25,10 +27,24 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
+	manifests "sigs.k8s.io/cluster-api/cmd/clusterctl/config"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/scheme"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/test"
 )
 
+func Test_VersionMarkerUpToDate(t *testing.T) {
```

Comment:
nit: we can use `g:=NewWithT(t)` and use gomega matchers here.
This is not a blocker though.